tests/regression/roles/sonic_ospf_area/tasks/preparation_tests.yaml

```diff
@@ -8,6 +9,6 @@
   vars:
     ansible_connection: httpapi
   sonic_ospf_area:
-    config: [] 
+    config: []
     state: deleted
-  ignore_errors: yes
+  failed_when: false
```

Comment:
This line is equivalent to the line it replaces. I realize that this is an ansible-lint fix, but it would be better to just delete this line. (We have recently discovered instances where test errors and sequencing problems are not being noticed due to subsequent ignoring of errors in preparation tasks.)

Please try deleting it and re-verifying that the regression test for this module passes.